File: common/src/test/java/com/hartwig/actin/clinical/util/ClinicalPrinterTest.java
package com.hartwig.actin.clinical.util;

import com.hartwig.actin.clinical.datamodel.TestClinicalFactory;

import org.junit.Test;

public class ClinicalPrinterTest {

    @Test
    public void canPrintClinicalRecords() {
        ClinicalPrinter.printRecord(TestClinicalFactory.createProperTestClinicalRecord());
        ClinicalPrinter.printRecord(TestClinicalFactory.createMinimalTestClinicalRecord());
    }
}

File: common/src/test/java/com/hartwig/actin/clinical/util/ClinicalPrinterTest.kt
package com.hartwig.actin.clinical.util

import com.hartwig.actin.clinical.datamodel.TestClinicalFactory.createMinimalTestClinicalRecord
import com.hartwig.actin.clinical.datamodel.TestClinicalFactory.createProperTestClinicalRecord
import com.hartwig.actin.clinical.util.ClinicalPrinter.Companion.printRecord
import org.junit.Test

class ClinicalPrinterTest {
    @Test
    fun canPrintClinicalRecords() {
        printRecord(createProperTestClinicalRecord())
        printRecord(createMinimalTestClinicalRecord())
    }
}